src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -113,11 +114,17 @@
     :fx [[:navigate-to-within-stack [:wallet-transaction-confirmation stack-id]]]}))
 
 (rf/reg-event-fx :wallet/get-suggested-routes
- (fn [{:keys [db now]} [amount]]
+ (fn [{:keys [db now]} [{:keys [amount]}]]
    (let [wallet-address          (get-in db [:wallet :current-viewing-account-address])
          token                   (get-in db [:wallet :ui :send :token])
```

Comment:
@briansztamfater - can you take a look at this again now?
this function is updated now to set `disabled-to-chain-ids` by taking all possible networks and removing the selected `bridge-to-chain-id`. Also sets the transaction type param using `(get-in db [:wallet :ui :send :type])` - this is also added for collectibles. For now it defaults to send asset but we can add this property to the other send asset flows too 👍